app/src/layouts/kanban/kanban.vue

```diff
@@ -156,6 +156,16 @@ const { t } = useI18n();
 const editDialogOpen = ref<string | number | null>(null);
 const editTitle = ref('');
 
+function getItemRoute(primaryKey: string | number) {
+	if (!props.collection) return '';
+
+	const route = props.collection.startsWith('directus_')
+		? props.collection.substring(9)
+		: `content/${props.collection}`;
+
+	return `/${route}/${primaryKey}`;
+}
+
```

Comment:
Would it make sense to put this into `@directus/utils` as well?

Reply:
Yes!

Reply:
Extracted it as an app util as it is app specific, with routing to `/content`.